scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -0,0 +1,180 @@
+# Lint as: python3
+"""
+This script preprocesses the covid19 data from Italy Department of Civil
+Protection for importing into Data Commons.
+"""
+
+import pandas as pd
+
+class PcmDpc:
+    """download the csv files and preprocess it for importing into
+    DataCommons"""
+    def preprocess(self):
+        """clean and save the CSV file"""
+        self.data = pd.read_csv(self.csvpath).drop(
+            columns=['note'])
+        self._translate()
```

Comment:
Can add an assert somewhere to make sure the csv had the expected columns.

Reply:
done.